src/app/profile/profile.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../../shared/services/user.service';
+import { User } from '../../shared/types/User';
+import { Router } from '@angular/router';
+import { OverlayService } from '../../shared/services/overlay.service';
+
+@Component({
+  selector: 'app-profile',
+  templateUrl: './profile.component.html',
+  styleUrls: ['./profile.component.scss']
+})
+export class ProfileComponent implements OnInit {
+  isPasswordModalOpen: boolean = true;
+  newEmail: string = '';
```

Comment:
create a new variable here called `isNameModalOpen` and initialize to false. Password modal should initialize to false as well, its only true atm for developing purposes